gc/base/TLHAllocationInterface.cpp

```diff
@@ -200,21 +200,17 @@ MM_TLHAllocationInterface::allocateObject(MM_EnvironmentBase *env, MM_AllocateDe
 
 	}
 
-	if (NULL != result) {
-		uintptr_t sizeInBytesAllocated = allocDescription->getContiguousBytes();
-		/* Increment by bytes allocated */
-		env->_traceAllocationBytes += sizeInBytesAllocated;
-
-		if (!allocDescription->isCompletedFromTlh()) {
+	if ((NULL != result) && !allocDescription->isCompletedFromTlh()) {
 #if defined(OMR_GC_OBJECT_ALLOCATION_NOTIFY)
-			env->objectAllocationNotify((omrobjectptr_t)result);
+		env->objectAllocationNotify((omrobjectptr_t)result);
 #endif /* OMR_GC_OBJECT_ALLOCATION_NOTIFY */
-			_stats._allocationBytes += sizeInBytesAllocated;
-			_stats._allocationCount += 1;
-		}
+		_stats._allocationBytes += allocDescription->getContiguousBytes();
+		_stats._allocationCount += 1;
 	}
-	env->_oolTraceAllocationBytes += (_stats.bytesAllocated() - _bytesAllocatedBase); /* Increment by bytes allocated */
 
+	uintptr_t sizeInBytesAllocated = (_stats.bytesAllocated(false) - _bytesAllocatedBase);
+	env->_oolTraceAllocationBytes += sizeInBytesAllocated; /* Increment by bytes allocated for Trace */
+	env->_traceAllocationBytes += sizeInBytesAllocated; /* Increment by bytes allocated for Hook */
```

Comment:
Remove the comments. They reference Trace and Hook, which is out of context for OMR. And they don't even comply with coding standards.